boards/beagle/beagleplay_cc1352/doc/index.rst

```diff
@@ -0,0 +1,192 @@
+.. _beagleplay_cc1352:
+
+BeaglePlay (CC1352)
+###################
+
+Overview
+********
+
+BeagleBoard.org BeaglePlay is an open hardware single board computer based on a TI Sitara AM6254
+quad-core ARM Cortex-A53 SoC with an external TI SimpleLink multi-standard CC1352P7 wireless MCU
+providing long-range, low-power connectivity.
+
+
+.. figure:: img/beagle_play.webp
+   :align: center
+   :width: 500px
+   :alt: BeagleBoard.org BeaglePlay
+
+   BeagleBoard.org BeaglePlay
+
+Hardware
+********
+
+* Processors
+
+  * TI Sitara AM6252 SoC
+
+    * 4x ARM Cortex-A53
+    * ARM Cortex-R5
+    * ARM Cortex-M4
+    * Dual-core 32-bit RISC Programmble Real-Time Unit (PRU)
+
+  * TI SimpleLink CC1352P7 Wireless MCU
+
+    * ARM Cortex-M4F programmable MCU
+    * ARM Cortex-M0+ software-defined radio processor
+
+* Memory
+
+  * 2GB DDR4
+  * 16GB eMMC flash
+  * I2C EEPROM
+
+* Wired connectivity
+
+  * Gigabit Ethernet (RJ45)
+  * Single-pair Ethernet with 5V/250mA PoDL output (RJ11)
+  * HDMI
+  * USB Type-A (host)
+  * USB Type-C (client/power)
+
+* Wireless connectivity
+
+  * TI WL1807 2.4GHz/5GHz WiFi
+  * BLE/SubG via CC1352P7
+
+* Expansion
+
+  * mikroBUS
+  * Grove
+  * QWIIC
+
+BeaglePlay ARM Cortex-A53 CPUs typically run Linux, while the CC1352P7 Cortex-M4 typically runs Zephyr.
+
+
+Supported Features
+==================
+
+The board configuration supports the following hardware features:
```

Comment:
*board target and put the name you use with west/cmake with \`\` before and after it